.github/workflows/ci.yml

```diff
@@ -32,6 +32,14 @@ jobs:
     - uses: actions/checkout@v2
     - name: Validate docker-compose.yml
       run: docker-compose -f docker-compose.yml config >/dev/null
+    - name: Set up Python 3.8
```

Comment:
I'm using `tox` in other repositories to run the test suite. Will be address in https://github.com/nlpsandbox/phi-deidentifier/issues/22.